src/MarkdownTextInput.web.tsx

```diff
@@ -156,7 +156,7 @@ const MarkdownTextInput = React.forwardRef<TextInput, MarkdownTextInputProps>(
     },
     ref,
   ) => {
-    const [isComposition, setIsComposition] = useState(false);
+    const compositionRef = useRef<boolean | null>(null);
```

Comment:
Does it work if we initialize the ref with `false`?
```suggestion
    const compositionRef = useRef<boolean>(false);
```